docs/installation-guide.md

```diff
@@ -0,0 +1,125 @@
+# AppX Installation Guide
+
+This guide will walk you though everything you need to add AppX to an existing Aragon DAO.
+
+---
+
+## TL;DR
+
+AppX has been published to the following locaionts:
+- [Local]()
+- [Rinkeby]()
+- [Mainnet]()
+
+To deploy to an organization you can use the [Aragon CLI](https://hack.aragon.org/docs/cli-intro.html).
+
+```sh
+aragon dao install <dao-address> AppX.open.aragonpm.eth --app-init-args <another-app-address> <another-app-address>
+```
+
+AppX must have the `SOMETHING_ROLE` permission on `AnotherApp` and the `STUFF_ROLE` permission on the `OtherApp`.
+
+---
+
+## Detailed Installation Guide
+
+---
+
+### 1. Deploy a fresh DAO
+
+This step is if you don't already have a DAO to install AppX on, or want to test it on a demo first.
+
+First, make sure that you have the [Aragon CLI](https://hack.aragon.org/docs/cli-intro.html) installed. Then run `aragon devchain` in a terminal. This should show you two Ethereum addresses. The first one has the most permissions and is used to execute commands on the Aragon CLI. Import the private key of that account into Metamask. Then head over to the [Rinkeby DAO launcher](rinkeby.aragon.org) and create a DAO with the democracy kit. Make sure that the Metamask account that is active is the first account created by your `aragon devchain`.
+
+Once your Democracy DAO is deployed (the voting params don't matter as you'll be the only one voting right now), go to the settings tab where you will find the addresses for the DAO and its apps. For legibility of subsequent commands will set bash environment variable for these addresses:
+
+```
+dao=0x6604f9fe9Db1D3F6a45d8F0ab79e8a4B05968816
+token=0x7F42cEB659B944cBB9F3D5ED637f66818C1bAcbf
+voting=0x41CA57d1e65Cdcd3A68A0e9f8E835F3a1FeDc655
+vault=0x04b46b9e0c1f893cA50Cb35F096d14dD946DEf95
+```
+
+---
+
+### 2. Install AppX to the DAO
+
+AppX has been published to the APM on Rinkeby at `AppX.open.aragonpm.eth`
```

Comment:
we are publishing the apps with a staging label for now, so it will be AppX-staging.open.aragonpm.eth

Reply:
I would say to leave it as it is, since we will most likely be publishing them without staging this week.